packages/mask/src/plugins/Savings/SNSAdaptor/SavingsForm.tsx

```diff
@@ -21,7 +21,8 @@ import {
     useWeb3,
     ZERO_ADDRESS,
 } from '@masknet/web3-shared-evm'
-import { FormattedCurrency, LoadingAnimation, TokenAmountPanel, TokenIcon } from '@masknet/shared'
+import { FormattedCurrency, LoadingAnimation, TokenIcon } from '@masknet/shared'
+import { TokenAmountPanel } from '../../../web3/UI/TokenAmountPanel'
```

Comment:
Those two `<TokenAmountPanel />` are almost the same thing. Could you please keep one and remove the other.

Reply:
https://github.com/DimensionDev/Maskbook/commits/develop/packages/mask/src/web3/UI/TokenAmountPanel.tsx
https://github.com/DimensionDev/Maskbook/commits/develop/packages/shared/src/UI/components/TokenAmountPanel/index.tsx

FYI @yanzhihong23